Sources/Fuzzilli/Util/JavaScriptExecutor.swift

```diff
@@ -89,13 +93,19 @@ public class JavaScriptExecutor {
         // Close stdin
         try inputPipe.fileHandleForWriting.close()
 
+        let environment = ProcessInfo.processInfo.environment.merging(
+            Dictionary(env, uniquingKeysWith: { _, new in new }),
```

Comment:
This shouldn't have to be a `Dictionary`?
```suggestion
            env,
```